content/en/smart-contracts/fundamentals/aggregators

```diff
@@ -0,0 +1,81 @@
+---
+title: "Aggregated Dealmaking"
+description: "Developers can aggregate their small deals, typically any deal <4GB in size, with other small deals, into one larger deal more attractive to Storage Providers."
+lead: "Developers can aggregate their small deals, typically any deal <4GB in size, with other small deals, into one larger deal more attractive to Storage Providers."
+draft: false
+images: []
+type: docs
+menu:
+  smart-contracts:
+    parent: "smart-contracts-fundamentals"
+    identifier: "aggregators"
+weight: 110
+toc: true
+aliases:
+
+---
+
+## Aggregation
+
+The design of Filecoin is to allow clients to store large amounts of data for a long period of time. Clients with small-scale data (<4 GiB) can aggregate their data, together with other small deals, into one large deal. For programmatic small-scale data storing (data >4GiB), a smart contract can be used to request for an aggregated storage deal to be made via FVM. 
+This article explains the programmatic aggregated dealmaking process, via the aggregator standard. We will refer to small-scale data as 'subpiece data'.
+
+Aggregation can be done off-chain or on-chain. For on-chain aggregation smart contracts, developers interact with them to submit subpiece data and to verify proof of storage for their data. The contract also allows for verification of provided PoDSI, which refers to the subpiece proof. For off-chain aggregation, developers interact with the Aggregator platform instead, although all required components are managed by the platform for a simpler UI to clients. Typically, off-chain aggregation requires a centralized platform to host various components. On-chain aggregator can typically function in a decentralized manner.
+
+The base interface requires 4 components:
+- Client who has data to upload
+- Aggregator platform (a type of storage solution) that clients can interact with to make a deal and retrieve proof of storage
+- Off-chain aggregation node to aggregate the subpiece data into a larger file
+- On-chain Aggregation smart contract that clients can interact with to request to make a deal and verify proof of storage
+
```

Comment:
Adding this workflow chart for generic concept of aggregation

<img width="838" alt="shapes (3)" src="https://github.com/filecoin-project/filecoin-docs/assets/113331491/9bf8c01b-9510-415f-8d4b-d45f2d1f4c7a">